tests/test_wallet.py

```diff
@@ -85,54 +87,124 @@ async def test_batch(self):
         nonce = await self.wallet.zk_provider.get_account_nonce(self.wallet.address())
 
         for i in range(3):
-            tr, sig = await self.wallet.build_transfer(
-                "0x21dDF51966f2A66D03998B0956fe59da1b3a179F",
-                amount=1, token=eth_token, fee=fee, nonce=nonce+i)
+            tr, sig = await self.wallet.build_transfer(self.receiver_address,
+                                                       amount=1,
+                                                       token=eth_token,
+                                                       fee=fee,
+                                                       nonce=nonce + i)
             trs.append(TransactionWithSignature(tr, sig))
         res = await self.wallet.send_txs_batch(trs)
         assert len(res) == 3
 
-    async def test_forced_exit(self):
-        tr = await self.wallet.forced_exit("0x21dDF51966f2A66D03998B0956fe59da1b3a179F",
-                                           "USDC")
+    async def test_build_batch_transfer(self):
+        nonce = await self.wallet.zk_provider.get_account_nonce(self.wallet.address())
+        builder = BatchBuilder.from_wallet(self.wallet, nonce)
+        for i in range(2):
+            builder.add_transfer(self.receiver_address, "ETH", Decimal(1))
+        build_result = await builder.build()
+        print(f"Total fees: {build_result.total_fees}")
+        trans = await self.wallet.zk_provider.submit_trx_batch_v2(build_result.transactions,
```

Comment:
Make sure that you wait for committing of one of the batch's transactions. Otherwise, we don't know if the transaction was executed

Reply:
I think It would be good to merge with the branch of Transaction Handler so. Tests will be more precise
Ok Sure. I will do

Reply:
@vyastrebovvareger I have merged the transaction handler branch into master a few days ago. Sorry for not letting you know right away. Could you please try now?

Reply:
Yeah. Just merging